test/util.ts

```diff
@@ -45,6 +50,7 @@ export const git = mocked(_git);
 
 // TODO: fix types, jest / typescript is using wrong overload (#7154)
 export const platform = mocked(partial<Required<Platform>>(_platform));
+export const scm = mocked(partial<Required<PlatformScm>>(_scm));
```

Comment:
```suggestion
export const scm = mocked(_scm);
```
i think we can use default here.